.gitignore

```diff
@@ -0,0 +1,132 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
```

Comment:
I wonder if this is working as I can see some `.pyc` files below.

Reply:
We think that was because they'd been added before the .gitignore. We've removed them now